webui/src/docs/templates/tou.js

```diff
@@ -0,0 +1,28 @@
+var React = require('react');
+
+module.exports = function(props) {
+  return (
+    <div className='term_of_use'>
+
+<h1>Terms of Use for EUDAT B2SHARE Service</h1>
+<p>By registering as a EUDAT B2SHARE service user (hereafter “B2SHARE”) you accept the following conditions of use:</p>
+<ul>
+<li>B2SHARE is made available to researchers and communities in order to share and preserve research data. B2SHARE should not be used for personal, recreational, political or commercial purposes.</li>
+<li>Use of the EUDAT services and resources is at your own risk. There is no guarantee that the B2SHARE service and resources will be available at any time or that it will suit any purpose. EUDAT is not liable to any compensation in case of lost data or loss of service.</li>
+<li>Logged information, including information provided by you for registration purposes, is used for administrative, operational, accounting, monitoring and security purposes only. This information may be disclosed, via secured mechanisms, only for the same purposes and only as far as necessary to other organizations cooperating with EUDAT. Although efforts are made to maintain confidentiality, no guarantees are given.</li>
+<li>All metadata stored in B2SHARE, except private information such as email addresses and phone numbers, must be publicly available.</li>
+<li>You must not use B2SHARE for any unlawful purpose and not attempt to breach, or breach or circumvent any administrative or security controls.</li>
+<li>You must respect the legal protection provided by copyright and licensing of software and data as well as intellectual property and confidentiality agreements. Do not store illegal copies of music, films, software or other media on B2SHARE.</li>
+<li>You must not store or access any potentially unlawful or clearly inappropriate content or data on B2SHARE</li>
+<li>You are accountable for the resource usage made by your account and you must therefore protect your account by good enough, difficult to guess passwords</li>
+<li>EUDAT is entitled to regulate, suspend or terminate your access to the service.</li>
+<li>You are liable for the consequences of violation of any of the conditions of use.</li>
+<li>Any violation being identified will be subject to corrective action, including possible termination of access, legal action, and criminal liability.</li>
+<li>To protect your data, B2SHARE relies on the best security practices. The Data Center where you data is stored, Datacenter CSC Kajaani, is certified by the ISO/IEC 27001:2005 standard for its information security management system.</li>
+</ul>
+<p>You will find further information related to B2SHARE service in the <a href="/help/Frequently-Asked-Question">FAQ</a></p>
```

Comment:
FAQ page should go away, see https://github.com/EUDAT-B2SHARE/b2share/issues/860